README.md

```diff
@@ -44,10 +47,61 @@ pip3 uninstall kompos --yes
 pip3 install --upgrade kompos
 ```
 
+### Nix integration
 
-### Nix versioning
+With kompos you can leverage [nix](https://nixos.org/nix/) to pin your
+infrastructure code (i.e terraform & helmfile releases) on a specific version.
+This enables you to finely control your deployments and use different
+infrastructure versions per environment, cluster etc.
+
+Enable the integration with the `--nix` flag with each command. Below are the
```

Comment:
why do we need `--nix` flag? Can't be enabled by default?

Reply:
The default behaviour is when you're doing development and you need to work with a locally checked-out repo.

Reply:
I see, would it make sense to rename it then to `--enable-remote-repos` ? nix is an impl details IIUC.